.github/workflows/release-plugins.yaml

```diff
@@ -0,0 +1,170 @@
+name: Backstage plugin release
+
+on:
+  pull_request:
+    branches:
+      - 'release-*'
+    paths:
+      - 'backstage/plugins/knative-event-mesh-backend/**'
+  push:
+    branches:
+      - 'release-*'
+      # Do not run on push to release-next branch. We don't want to publish the plugin on every push to release-next.
+      # It might be okay to do so, but sobranch tool doesn't work well with release-next branch name.
+      - '!release-next'
+# Ideally, we would want to only publish the plugin on NPM when there's an actual code change in the plugin.
+# However, using the path filters block workflow execution on initial branch creation.
+# So, let's publish on every push to a release branch.
+# see https://github.com/openshift-knative/backstage-plugins/pull/15#issuecomment-2333324099
+#    paths:
+#      - 'backstage/plugins/knative-event-mesh-backend/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            event_mesh_plugin:
+              - 'backstage/plugins/knative-event-mesh-backend/**'
+
+      - name: Setup Golang
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: Checkout openshift-knative/hack
+        uses: actions/checkout@v4
+        with:
+          repository: 'openshift-knative/hack'
+          ref: 'main'
+          path: ./src/github.com/openshift-knative/hack
+
+      - name: Install sobranch
+        run: |
+          go install ./cmd/sobranch
+        working-directory: ./src/github.com/openshift-knative/hack
+
+      - name: Setup release name for release branch push
+        # do not run the step on PRs against release-next branch. run it on:
+        # PRs against release-* branches, push to release-* branches, and creation of release-* branches
+        if: github.event_name == 'create' || steps.changes.outputs.event_mesh_plugin == 'true'
```

Comment:
where did you see this syntax? in SO we're using https://github.com/openshift-knative/serverless-operator/blob/d5c4121801f91f2dbcaf3ccf4ae793e38266cd42/.github/workflows/bump.yaml#L16 so it's a little different here

Reply:
I am ok with that syntax as well, as long as it works.